privacy/__manifest__.py

```diff
@@ -2,12 +2,13 @@
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
     'name': 'Data Privacy and Protection',
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Data Protection',
     'summary': 'Provides data privacy and protection features '
                'to comply to regulations, such as GDPR.',
     'author': "Eficent, "
               "Tecnativa, "
+              "Nicolas JEUDY, "
```

Comment:
These little changes don't grant you as co-author. Please only add you on contributors.

Reply:
ok, is there a place to now when you become author ?

Reply:
There's no exact metric, and it's valued by reviewers on each specific case. A migration hardly will be a co-author task, as most of the work is to just test things. If the migration requires to change a significant part of the module, then can be qualified for it. Also introducing a new big feature, doing a big refactoring, or adding a big bunch of tests are other promoting tasks.